src/reports/models/reportsManager.js

```diff
@@ -65,9 +63,10 @@ module.exports.deleteReport = async (testId, reportId) => {
     await databaseConnector.deleteReport(testId, reportId);
 };
 
-module.exports.postReport = async (testId, reportBody) => {
-    const startTime = new Date(Number(reportBody.start_time));
-    const job = await jobConnector.getJob(reportBody.job_id);
+module.exports.postReport = async(runId, testId, jobId, startTime) => {
+    const reportId = runId;
+    const test = await testsManager.getTest(testId);
+    const job = await jobConnector.getJob(jobId);
```

Comment:
this line is also not needed for the same reason as the getTest ^

Reply:
done